hadoop-tools/hadoop-aws/src/test/resources/log4j.properties

```diff
@@ -57,6 +57,8 @@ log4j.logger.org.apache.hadoop.ipc.Server=WARN
 #log4j.logger.org.apache.hadoop.fs.s3a.Listing=INFO
 # Log S3Guard classes
 #log4j.logger.org.apache.hadoop.fs.s3a.s3guard=DEBUG
+# if set to debug, this will log the PUT/DELETE operations on a store
+log4j.logger.org.apache.hadoop.fs.s3a.s3guard.Operations=DEBUG
```

Comment:
we may don't want to log on debug for everybody.

Reply:
for test run's it's actually quite interesting. But yes, I'll turn it off